CadRevealFbxProvider/FbxSdkWrapper.cs

```diff
@@ -13,7 +13,7 @@ public class FbxSdkWrapper : IDisposable
     public FbxSdkWrapper()
     {
         CreateSdk();
-        _isValidSdk = AssertFbxSdkVersion();
+        _isValidSdk = AssertFbxSdkVersion("2020.3.2");
```

Comment:
"2020.3.2" is a magic string here. Extract it to a constant CurrentExpectedSdkVersion and consider adding xml comment that this is to ensure the fbx dll is in sync, so if the dll has a newer version this stringn should be updated.